ex14.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
+
+"""combine a one and a two dimensional array together and display their elements"""
+one_dimensional = np.arange(4)
+two_dimensional = np.arange(8).reshape(2, 4)
+for first_element, second_element in np.nditer([one_dimensional, two_dimensional]):
+    print("%d:%d" % (first_element, second_element), )
```

Comment:
This is a strange way of printing in python.
This is clearly copied.
The least you can do is change this into a function.